VulkanHppGenerator.cpp

```diff
@@ -515,7 +516,7 @@ void VulkanHppGenerator::generateVulkanToStringHppFile() const
 
 #include <vulkan/vulkan_enums.hpp>
 
-#if ( ( 20 <= VULKAN_HPP_CPP_VERSION ) && __has_include( <format> ) )
+#if ( ( 20 <= VULKAN_HPP_CPP_VERSION ) && __cpp_lib_format )
```

Comment:
the check for cpp 20 is actually not necessary. `#if __cpp_lib_format` would be enough

Reply:
That's right. That is, please remove the check against `VULKAN_HPP_CPP_VERSION`.